### PR TITLE
Sb homework08

### PR DESCRIPTION
**Pull request by foggydawning**

### CODE REVIEW

File: SB_homework08/main_tasks.playground/Contents.swift [hunk 1]
```diff
@@ -1,3 +1,92 @@
...
-var greeting = "Hello, playground"
+// task №1: create getAgeComparingString func
+// task №2: create getInfo func
+struct Person {
+    var name: String
+    var age: Int
+
+ 
+    func getAgeComparisonString(_ p: Person) -> String {
+        // A function for comparing the age of two Person
```
Comment (heartsker) [inline SB_homework08/main_tasks.playground/Contents.swift:13]: Better to use enum in this case

Comment (heartsker) [inline SB_homework08/main_tasks.playground/Contents.swift:13]: Better to use enum in this case

File: SB_homework08/main_tasks.playground/Contents.swift [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+        
+        return "\(p.name) \(result)"
+    }
+    
+    func getInfo() -> String{
+        "\(self.name) \(self.age) \(checkRussianWordYear())"
+    }
+    
+    private func checkRussianWordYear() -> String{
+
```
Comment (heartsker) [inline SB_homework08/main_tasks.playground/Contents.swift:37]: Could be done easier by using only switch with case ... where ...

File: SB_homework08/main_tasks.playground/Contents.swift [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+    }
+}
+
+// task №3: to create class Hero with lifeCount property and hit() method
+// task №4: to add to the Hero Class isAlive() property
+// task №5: convert the var lifeCount to the private var
+
+class Hero {
+    private var lifeCount: Int
+
```
Comment (heartsker) [inline SB_homework08/main_tasks.playground/Contents.swift:64]: Do not use additional properties.

File: SB_homework08/main_tasks.playground/Contents.swift [hunk 4]
```diff
@@ -1,3 +1,92 @@
...
+    
+    func hit(){
+        self.lifeCount -= basicRecivedDamage
+    }
+}
+
+// task №6: create the class SuperHero from the class Hero
+
+class SuperHero: Hero {
+
```
Comment (heartsker) [inline SB_homework08/main_tasks.playground/Contents.swift:86]: Follow the task. Method hit should be overwritten.

File: SB_homework08/main_tasks.playground/Contents.swift [hunk 5]
```diff
@@ -1,3 +1,86 @@
...
+        }
+        return result
+    }
+}
+
+// task №3: to create class Hero with lifeCount property and hit() method
+// task №4: to add to the Hero Class isAlive() property
+// task №5: convert the var lifeCount to the private var
+class Hero {
+    private var lifeCount: Int
```
[the remark this answers is not in the document]
Comment (foggydawning) [reply SB_homework08/main_tasks.playground/Contents.swift:61]: simplified 😊

**Pull request merged**